index resume.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+ <html>
+    <head>
+        <title>Soma Ward's Resume</title>
+  <hl>SOMA WARD'S RESUME</hl>
+     <p>
+         <ul>
+            <li>Objectives</li>
+              <ol>To provide superior websites and apps at a reasonable price</ol>
+              <ol>Contribute to a strong team of progammers at a great company</ol>
+              <ol>Assist the young people in the community in learning to Code</ol>
+            <li>Work Experience</li>
+            <ol>Diesel Mechanic</ol>
+             <ol>HVAC Technician</ol>
+            <li>Education</li>
+            <li>Skills</li>
+               <ol>General Laborer</ol>
+                <ol>Plumbing Technician</ol>
+                <ol>Minor Mig-welding</ol>
+               
+               
+            <li>Profile Image</li>
+              <img src="20201028_223834.jpg"height=9% width=9%>
+            <li>Awards or Certificates</li>
+              <ol>Certified Diesel Mechanic</ol>
+              <ol>Certified HVAC Technician</ol>
+              
+             
+               <li><a href="https:www.linkedin.com/in/soma-ward-b6a9a8141">Linkedin Profile Address</li>
+              
+            
+
+         </ul>
+
+
+     </p>
+          
+    
+    </head>
+    <body>
+    
+        
+
+            
+        
+        
+
+
+    </body>
+
+
+</html>
```

Comment:
Indentation and styling are not consistent (-2)
Missing Part 2 (-50)

---------------------------------------
44/100
Go ahead and focus on part two now